codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerServiceGeneratorV2.kt

```diff
@@ -321,6 +321,79 @@ class ServerServiceGeneratorV2(
 
         rustTemplate(
             """
+            /// The [`$serviceName`] is the place where you can register
```

Comment:
> The [`$serviceName`] is the place where you can register

This seems a little confusing because it's the builder where you register handlers.